flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -257,20 +257,25 @@
 		key("taskmanager.memory.process.size")
 			.memoryType()
 			.noDefaultValue()
-			.withDescription("Total Process Memory size for the TaskExecutors. This includes all the memory that a"
-				+ " TaskExecutor consumes, consisting of Total Flink Memory, JVM Metaspace, and JVM Overhead. On"
-				+ " containerized setups, this should be set to the container memory.");
+			.withDescription("Total Process Memory size for the TaskExecutors. This includes all the memory that a "
+				+ "TaskExecutor consumes, consisting of Total Flink Memory (configured with "
+				+ "'taskmanager.memory.flink.size'), JVM Metaspace, and JVM Overhead. On "
+				+ "containerized setups, this should be set to the container memory."
```

Comment:
I would put the reference to `flink.size` at the end of the description similar to what we are doing in the description of `TOTAL_FLINK_MEMORY`.